openslides/users/static/js/users/site.js

```diff
@@ -850,76 +826,88 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
 
 .controller('GroupListCtrl', [
     '$scope',
+    '$http',
+    'operator',
     'Group',
-    function($scope, Group) {
+    'permissions',
+    function($scope, $http, operator, Group, permissions) {
         Group.bindAll({}, $scope, 'groups');
+        $scope.permissions = permissions;
+
+        $scope.apps = [];
+        var lastApp = '';
+
+        // Create the main clustering with appname->permissions
+        angular.forEach(permissions, function(perm) {
+            var permissionApp = perm.value.split('.')[0]; // get appname
+            var translations = { "agenda": "Agenda",
+                                 "assignments": "Elections",
```

Comment:
You should use a (translateable) verbose name defined by each app, e.g.:
https://github.com/OpenSlides/OpenSlides/blob/e688a345c088bf27fdf448a53ba623258a807ee3/openslides/assignments/static/js/assignments/base.js#L169
You still need a plural form of this string...